addons/l10n_ar_stock/i18n/es_419.po

```diff
@@ -0,0 +1,313 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* l10n_ar_stock
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 18.4a1+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2025-06-03 16:01+0000\n"
+"PO-Revision-Date: 2025-06-03 16:01+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: l10n_ar_stock
+#. odoo-python
+#: code:addons/l10n_ar_stock/models/stock_picking_type.py:0
+msgid "%(company)s Sequence %(name)s"
+msgstr "Secuencia %(name)s de %(company)s"
+
+#. module: l10n_ar_stock
+#. odoo-python
+#: code:addons/l10n_ar_stock/models/stock_picking_type.py:0
+msgid ""
+"%(sequence_number)s is not a valid sequence number. Sequence numbers should "
+"contain exactly 8 digits (e.g. 00012345)."
+msgstr ""
+"%(sequence_number)s no es un número de secuencia válido. Los números de secuencia
+deben contener exactamente 8 dígitos (por ejemplo, 00012345)."
```

Comment:
```suggestion
"%(sequence_number)s no es un número de secuencia válido. Los números de secuencia"
"deben contener exactamente 8 dígitos (por ejemplo, 00012345)."
```

Reply:
Oh ,I see! 
Sorry, it was my fault, I did the translation and I did not catch that error :/

Reply:
All of the other changes look good, so solving this should leave everything working OK!
Thanks!